test/test_device.py

```diff
@@ -535,6 +535,35 @@ def test_display_address_descriptor(self):
         self.assertIn('code', result)
         self.assertEqual(result['code'], -7)
 
+    def test_display_address_multisig(self):
+        if 'trezor' in self.full_type:
```

Comment:
If we could upload, say, m/5 multisig wallet to the coldcard containing xpubs from each simulator then I wouldn't need this if-statement.